cypress/e2e/calendar/check-view.cy.js

```diff
@@ -0,0 +1,71 @@
+import { Button, MultiColumnListCell, Pane, including, Link } from '../../../interactors';
+import {
+  checkCalendarFields, checkExpandButton, checkMenuAction
+} from '../../support/fragments/calendar/calendar-info-pane';
+import calendarFixtures from '../../fixtures/calendar_e2e_fixtures';
+
+const testServicePoint = calendarFixtures.servicePoint;
+const testCalendar = calendarFixtures.calendar;
+
+
+describe('Checking the view of calendar on "All Calendars" tab', () => {
+  let testCalendarResponse;
+
+  before(() => {
+    // login and open calendar settings
+    cy.loginAsAdmin();
+
+    // reset db state
+    cy.deleteServicePoint(testServicePoint.id, false);
+
+    // create test service point and calendar
+    cy.createServicePoint(testServicePoint, (response) => {
+      testCalendar.assignments = [response.body.id];
+
+      cy.createCalendar(testCalendar, (calResponse) => {
+        testCalendarResponse = calResponse.body;
+      });
+    });
+  });
+
+
+  beforeEach(() => {
+    cy.openCalendarSettings();
+    cy.do(Pane('Calendar').find(Link('All calendars')).click());
+  });
+
+  after(() => {
+    // delete test calendar
+    cy.deleteCalendar(testCalendarResponse.id);
+  });
+
+
+  it('should check that the appropriate actions menu exists in the "All calendars" tab', () => {
```

Comment:
`it('{test case id from test rail} {test case name from test rail} (vega)', { tags: [TestTypes.smoke, devTeams.vega] }, () => {`